app/policies/application_policy.rb

```diff
@@ -2,27 +2,33 @@ class ApplicationPolicy
   attr_reader :person, :record
 
   def initialize(person, record)
-    @person = person
+    @person = person || Person.new
```

Comment:
This seems weird to me. We'll never end up creating this record. @brendandeere what do you think about this?